Tests/Document/Change/ChangeTests.swift

```diff
@@ -0,0 +1,86 @@
+
+        let object2Ticket = TimeTicket(lamport: 4, delimiter: 0, actorID: actorId)
+        let object2 = CRDTObject(createdAt: object2Ticket)
+        let b1 = Primitive(value: .string("b1"), createdAt: TimeTicket(lamport: 5, delimiter: 0, actorID: actorId))
+        object2.set(key: "k-b1", value: b1)
+
+        rootObject.set(key: "k-a3", value: object2)
+
+        let object3 = CRDTObject(createdAt: TimeTicket(lamport: 6, delimiter: 0, actorID: actorId))
+        let c1 = Primitive(value: .string("c1"), createdAt: TimeTicket(lamport: 7, delimiter: 0, actorID: actorId))
+        object3.set(key: "k-c1", value: c1)
+
+        let setOperation = SetOperation(key: "k-d2", value: object3,
+                                        parentCreatedAt: object2.createdAt,
+                                        executedAt: TimeTicket(lamport: 8, delimiter: 0, actorID: self.actorId))
+
+        let changeID = ChangeID(clientSeq: 1, lamport: 2, actor: self.actorId)
+
+        let target = Change(id: changeID, operations: [setOperation])
+
+        XCTAssertEqual(target.getOperations()[0].executedAt.getStructureAsString(), "8:actor-1:0")
+
+        target.setActor("actor-2")
+
+        XCTAssertEqual(target.getOperations()[0].executedAt.getStructureAsString(), "8:actor-2:0")
+    }
+
+    func test_can_execute() throws {
+        let rootObject = CRDTObject(createdAt: TimeTicket(lamport: 1, delimiter: 0, actorID: actorId))
+        let a1 = Primitive(value: .string("a1"), createdAt: TimeTicket(lamport: 2, delimiter: 0, actorID: actorId))
+        rootObject.set(key: "k-a1", value: a1)
+
+        let object2Ticket = TimeTicket(lamport: 4, delimiter: 0, actorID: actorId)
+        let object2 = CRDTObject(createdAt: object2Ticket)
+        let b1 = Primitive(value: .string("b1"), createdAt: TimeTicket(lamport: 5, delimiter: 0, actorID: actorId))
+        object2.set(key: "k-b1", value: b1)
+
+        rootObject.set(key: "k-a3", value: object2)
+
+        let root = CRDTRoot(rootObject: rootObject)
+
+        let object3 = CRDTObject(createdAt: TimeTicket(lamport: 6, delimiter: 0, actorID: actorId))
+        let c1 = Primitive(value: .string("c1"), createdAt: TimeTicket(lamport: 7, delimiter: 0, actorID: actorId))
+        object3.set(key: "k-c1", value: c1)
+
+        let setOperation = SetOperation(key: "k-d2", value: object3,
+                                        parentCreatedAt: object2.createdAt,
+                                        executedAt: TimeTicket(lamport: 8, delimiter: 0, actorID: self.actorId))
+
+        let changeID = ChangeID(clientSeq: 1, lamport: 2, actor: self.actorId)
+
+        let target = Change(id: changeID, operations: [setOperation])
+
+        XCTAssertEqual(root.toSortedJSON(), "{\"k-a1\":\"a1\",\"k-a3\":{\"k-b1\":\"b1\"}}")
+
+        try target.execute(root: root)
+
+        XCTAssertEqual(root.toSortedJSON(), "{\"k-a1\":\"a1\",\"k-a3\":{\"k-b1\":\"b1\",\"k-d2\":{\"k-c1\":\"c1\"}}}")
```

Comment:
Can this line be expressed as the below(without escape characters)?

```suggestion
        XCTAssertEqual(root.toSortedJSON(), """{"k-a1":"a1","k-a3":{"k-b1":"b1","k-d2":{"k-c1":"c1"}}}""")
```
https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html

Reply:
I was worried about it too.